pages/byTheNumbers.tsx

```diff
@@ -0,0 +1,76 @@
+import StatBox from './stat-box';
+// todo:
+// 1. install inter font(tried and failed)
+// 2. card in each grid should be narrower
+// 3. alignment on three bottom cards
+export default function ByTheNumbers() {
+    return (
+        <div className="flex flex-col p-8  justify-center items-center gap-y-3.5 md:w-full md:grid md:grid-cols-3 md:grid-rows-2 md:grid-flow-col md:gap-6 lg:gap-14 md:justify-items-center md:items-center md:p-16  bg-spec-lightTurquiose mt-6
+        ">
+            <div className="block md:hidden text-base font-medium opacity-40 text-black  font-montserrat text-left mr-auto pl-10">
+                <p>BY THE NUMBERS</p>
+            </div>
+
+            <StatBox
+                className=" border border-spec-turquiose md:text-base md:border-none row-start-1 col-start-1 row-span-2 font-normal rounded-md bg-spec-white font-dmserif text-spec-turquiose "
+
+                height="h-96"
+                width="w-full"
+            >
+                <p className='text-7xl md:text-6xl lg:text-8xl'>90%</p>
+                <p className='text-base text-black m-6 font-montserrat font-normal leading-7 md:text lg:text-lg'>
+                    of individuals inside the organization come from diverse backgrounds; that might include visible/invisible
+                    disabilities, LGBTQ+, women, people of color.
+                </p>
+            </StatBox>
+
+            <div className="hidden md:block row-start-1 col-start-2 row-span-2 col-span-2">
+                <StatBox
+                    className='text-left rounded-md font-dmserif text-spec-turquiose '
+                    height="h-full"
+                    width="w-full"
+                >
+                    <h2 className="text-lg font-medium opacity-40 text-black text-left font-montserrat mb-8">
```

Comment:
Good Work! Per the design, we need to make a slight adjustment to this StatBox. RIght now the text is stretching across the screen and we want it to take up part of the screen to match the figma. You can achieve this a a couple of ways. One way is to wrap this header and paragrah in a div and set a fraction for a width (3/5)
<img width="307" alt="Screenshot 2023-04-10 at 11 56 38 AM" src="https://user-images.githubusercontent.com/66931010/230968003-0959fb08-967d-4ec1-8470-56f6323e7370.png">

Reply:
Looks great!

Reply:
One last thing, the Figma design has changed slightly. I caught this while I was reviewing your PR. The mobile design should include the paragraph that starts with "We are striving to ...". I think you'll just need to add that back in for mobile.